extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -508,28 +563,56 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 		this._fileShareContainer.updateCssStyles({ 'display': (containerType === NetworkContainerType.FILE_SHARE) ? 'inline' : 'none' });
 		this._blobContainer.updateCssStyles({ 'display': (containerType === NetworkContainerType.BLOB_CONTAINER) ? 'inline' : 'none' });
 		this._networkShareContainer.updateCssStyles({ 'display': (containerType === NetworkContainerType.NETWORK_SHARE) ? 'inline' : 'none' });
-		this._networkShareLocationText.updateProperties({
-			required: containerType === NetworkContainerType.NETWORK_SHARE
+		this._networkShareLocations.forEach((inputBox) => {
+			inputBox.updateProperties({
+				required: containerType === NetworkContainerType.NETWORK_SHARE
+			});
 		});
 		this._windowsUserAccountText.updateProperties({
 			required: containerType === NetworkContainerType.NETWORK_SHARE
 		});
 		this._passwordText.updateProperties({
 			required: containerType === NetworkContainerType.NETWORK_SHARE
 		});
-
-		this._networkShareLocationText.validate();
+		this._networkShareLocations.forEach((inputBox) => {
+			inputBox.validate();
+		});
 		this._windowsUserAccountText.validate();
 		this._passwordText.validate();
 		this._networkShareContainerSubscriptionDropdown.validate();
 		this._networkShareContainerStorageAccountDropdown.validate();
 		this._blobContainerSubscriptionDropdown.validate();
 		this._blobContainerStorageAccountDropdown.validate();
-		this._blobContainerBlobDropdown.validate();
+		this._blobContainerDropdowns.forEach((dropdown) => {
+			dropdown.validate();
+		});
 		this._fileShareSubscriptionDropdown.validate();
 		this._fileShareStorageAccountDropdown.validate();
-		this._fileShareFileShareDropdown.validate();
+		this._fileShareDropdowns.forEach((dropdown) => {
```

Comment:
FYI you don't need to add the parens when you have just a single parameter without its type